CONTRIBUTING.md

```diff
@@ -331,6 +332,33 @@ flog --bytes $((100 * 1024 * 1024)) > sample.log
 
 This will create a `100MiB` sample log file in the `sample.log` file.
 
+#### Tips and Tricks
+
+If you are developing a particular component and want to quickly iterate on unit
+tests related only to this component, the following approach can reduce waiting times:
+
+1. Install [cargo-watch](https://github.com/passcod/cargo-watch).
+2. (Only for GNU/Linux) Install LLVM 9 (for example, package `llvm-9` on Debian)
+   and set `RUSTFLAGS` environment variable to use `lld` as the linker:
+
+   ```sh
+   export RUSTFLAGS='-Clinker=clang-9 -Clink-arg=-fuse-ld=lld'
```

Comment:
Looks like for me that'd be `RUSTFLAGS='-Clinker=clang-9 -Clink-arg=-fuse-ld=lld'`. Could you provide a tiny additional explainer what those mean so that users can troubleshoot if `lld` is `lld-9` on their system?

Reply:
> Looks like for me that'd be `RUSTFLAGS='-Clinker=clang-9 -Clink-arg=-fuse-ld=lld'`.

But what is the difference?

> Could you provide a tiny additional explainer what those mean so that users can troubleshoot if lld is lld-9 on their system?

The `lld` is `lld-9` because `clang` is called as `clang-9`. But the `lld` version can be checked by running `clang-9 -fuse-ld=lld --verbose <a file with hello world in C>.c`. However, I removed mentioning of the version to avoid confusion.

Reply:
Sorry, I didn't check what I pasted, I meant: `RUSTFLAGS='-Clinker=clang-9 -Clink-arg=-fuse-ld=lld-9'`. `lld` is not available on my system.

Reply:
I agree, if `lld-9` is not installed, but `lld` (older version) in `PATH`, then `clang-9` can pick the wrong version. So I just removed the version to make it more future-proof and avoid confusion, older versions should work oto.